pkg/manager/member/template.go

```diff
@@ -220,7 +220,7 @@ fi
 
 # Use HOSTNAME if POD_NAME is unset for backward compatibility.
 POD_NAME=${POD_NAME:-$HOSTNAME}
-ARGS="--pd={{ .PDAddress }} \
+ARGS="--pd={{ .Scheme }}://${CLUSTER_NAME}-pd:2379 \
```

Comment:
The `CLUSTER_NAME` may be the tc name or the reference tc name in `spec.cluster`.